TwoPage/src/main/java/com/microsoft/device/display/samples/twopage/ui/theme/Theme.kt

```diff
@@ -5,17 +5,20 @@ import androidx.compose.material.MaterialTheme
 import androidx.compose.material.darkColors
 import androidx.compose.material.lightColors
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
 
 private val DarkColorPalette = darkColors(
-    primary = Purple200,
-    primaryVariant = Purple700,
-    secondary = Teal200
+    primary = lightBlue,
+    primaryVariant = lightBlue,
+    secondary = Teal200,
+    onPrimary = Color.White,
 )
 
 private val LightColorPalette = lightColors(
-    primary = Purple500,
-    primaryVariant = Purple700,
-    secondary = Teal200
+    primary = lightBlue,
+    primaryVariant = lightBlue,
+    secondary = Teal200,
+    onPrimary = Color.White,
```

Comment:
same for TwoPage, I think the bundle app version only has the light color palette